src/components/IstioWizards/WizardActions.ts

```diff
@@ -1387,9 +1387,14 @@ export const buildSidecar = (name: string, namespace: string, state: SidecarStat
   return sc;
 };
 
-export const buildNamespaceInjectionPatch = (enable: boolean, remove: boolean): string => {
+export const buildNamespaceInjectionPatch = (enable: boolean, remove: boolean, revision: string | null): string => {
```

Comment:
```
revision: string | null
```
Do you mean 'null' or 'undefined' ?

Typically you can use the expression:

```
revision?: string
```

For undefined/undeclared fields, which I think is what you may want to express here.

Reply:
this function can be called from two places, one with revision param value, another without that param at all

Reply:
Ah, and I see that the "null" value is used in the patch to remove the parameter.
Ok.